bigquery_etl/cli/query.py

```diff
@@ -419,13 +418,7 @@ def info(ctx, name, sql_dir, project_id, cost, last_updated):
 
     query_files = paths_matching_name_pattern(name, sql_dir, project_id)
     if query_files == []:
-        # run SQL generators if no matching query has been found
-        ctx.invoke(
-            generate_all,
-            output_dir=ctx.obj["TMP_DIR"],
-            ignore=["derived_view_schemas", "stable_views"],
-        )
-        query_files = paths_matching_name_pattern(name, ctx.obj["TMP_DIR"], project_id)
+        raise click.ClickException(f"No query matching `{name}` was found.")
```

Comment:
We could do that, but IMO automatically running all SQL generators when a query isn't found is generally undesirable.  Back when there were fewer SQL generators and they ran quickly that probably made sense as a convenience, but now I think it's a net negative (e.g. in a local `bqetl query` command you make a typo in the query name and now it's going off and trying to generate all the things and polluting your working copy).

Reply:
I can see advantages for both sides: users don't need to know that a query is generated if they want to run a query, because it's implicitly taken care of vs having to wait a while for the generation to finish.

I think for fixing this we shouldn't change the generation behaviour for now. That might be something for a follow-up PR

Reply:
OK, I reverted the changes removing auto-running SQL generation.